pinecone/sparse/splade/splade_vector_generation.ipynb

```diff
@@ -0,0 +1,8784 @@
+{
```

Comment:
<p>"[F]or more details on that[,] check[ ]out this notebook."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>